documentation_builder/requirements.txt

```diff
@@ -1,5 +1,6 @@
-Sphinx~=2.2
+Sphinx~=5.3
 sphinxcontrib-napoleon
 sphinx-autoapi
 nbsphinx>=0.2.4
 ipykernel
+appdirs
```

Comment:
Why is this new dependency needed here?

Reply:
It isn't. I didn't realize that with the new yaml it would ignore the config in the RTD web interface and got a bunch of errors for missing imports. I'll remove it.